charts/lighthouse/Makefile

```diff
@@ -35,7 +35,7 @@ release: clean
 	helm lint
 	helm init --client-only
 	helm package .
-	curl --fail -u $(CHARTMUSEUM_USER):$(CHARTMUSEUM_PASS) --data-binary "@$(NAME)-$(shell sed -n 's/^version: //p' Chart.yaml).tgz" $(CHART_REPO)/api/charts
+	@curl --fail -u $(CHARTMUSEUM_USER):$(CHARTMUSEUM_PASS) --data-binary "@$(NAME)-$(shell sed -n 's/^version: //p' Chart.yaml).tgz" $(CHART_REPO)/api/charts
```

Comment:
+1, I think we can thin out the rules in these `Makefiles` altogether, but I can create a PR for that.